library/tmpdir/dir/mktmpdir_spec.rb

```diff
@@ -38,7 +40,7 @@
     Dir.mktmpdir do |path|
       @tmpdir = path
       called = true
-      path.should =~ /^\/tmp\//
+      path.should =~ /^#{real_tmp_root}/
```

Comment:
`path.start_with?(@real_tmp_root).should be_true` (too bad we don't have a start_with matcher, I should add that).